spec/support/delayed_job.rb

```diff
@@ -81,7 +81,7 @@ def query_jobs(object, method_name)
         .where("handler LIKE ?", "%\nmethod_name: :#{method_name}\n%")
 
       scope.select do |job|
-        return object == job.payload_object.object unless object.respond_to?(:matches?)
+        next object == job.payload_object.object unless object.respond_to?(:matches?)
```

Comment:
What does this do?

Reply:
`return` was terminating the whole method and returning a boolean value as a AR scope. `next` terminates just a single block invocation and yields said boolean to the iterating method (`any?`) allowing further iteration.